### PR TITLE
Add configuration to disable bootstrap of admin account

### PR DESCRIPTION
**Pull request by marioschlipf**
I've came across a challenge when using this testcontainer in combination with importing realm configuration: Since the env variable for bootstrapping admin accounts are hardcoded, I cannot automatically import a full Keycloak configuration:

```
            withEnv("KC_BOOTSTRAP_ADMIN_USERNAME", adminUsername);
            withEnv("KC_BOOTSTRAP_ADMIN_PASSWORD", adminPassword);
```

Auto-importing configuration like so will fail, if the realm contains an admin user:

```
new KeycloakContainer()
            .withRealmImportFiles(MASTER_REALM_USERS_JSON))
```              

The use case for this is that one might want to export and import a 1:1 export of a running keycloak installation for test reasons.

I've added a flag to disable this behavior, see the test.

### CODE REVIEW

File: src/main/java/dasniko/testcontainers/keycloak/ExtendableKeycloakContainer.java
Comment (dasniko) [inline src/main/java/dasniko/testcontainers/keycloak/ExtendableKeycloakContainer.java:169]: Please add a space before the brace to meet existing code formatting in this repo.

File: src/main/java/dasniko/testcontainers/keycloak/ExtendableKeycloakContainer.java
Comment (dasniko) [inline src/main/java/dasniko/testcontainers/keycloak/ExtendableKeycloakContainer.java:524]: Perhaps the method should be named `withBootstrapAdminDisabled()`!? WDYT?
IMHO it's easier to read, as "without" is too similar with "with" and it's not clear enough that with this setting something will be disabled.

File: src/test/java/dasniko/testcontainers/keycloak/KeycloakContainerTest.java
Comment (dasniko) [inline src/test/java/dasniko/testcontainers/keycloak/KeycloakContainerTest.java:37]: Just something minor: "master-realm-with-admin-user" would be a better naming fit, IMHO. Both, for constant and file name.

**Review by dasniko — CHANGES_REQUESTED** [review]
Thanks for this PR, generally LGTM and makes somehow sense with the new behavior.

There are some small issues, mentioned directly at the code, please review and fix it.
Additionally, there is some documentation missing. Please add a paragraph in the README file.

Thanks.

File: src/main/java/dasniko/testcontainers/keycloak/ExtendableKeycloakContainer.java
Comment (marioschlipf) [reply src/main/java/dasniko/testcontainers/keycloak/ExtendableKeycloakContainer.java]: Amended

File: src/main/java/dasniko/testcontainers/keycloak/ExtendableKeycloakContainer.java
Comment (marioschlipf) [reply src/main/java/dasniko/testcontainers/keycloak/ExtendableKeycloakContainer.java]: Well spotted, done

**Comment by marioschlipf** [comment]
> Additionally, there is some documentation missing. Please add a paragraph in the README file.

Thanks, comments addressed and documentation amended. I moved the `Initial admin user credentials` section above the `Realm Import` so the added paragraph makes sense.

Greetings from a fellow Darmstaedter

**Comment by marioschlipf** [comment]
Thanks for the approval, I don't have the rights to merge though

**Comment by dasniko** [comment]
Yes, and that's good as this is my repo and would yield to a security issue if anybody would have merge rights.

Running the tests take some time and I don't have always this time to wait, so reviewing and merging are separate tasks at different times.
And before you ask - merging a new feature does not necessarily result in a new release. Be patient 🧘

**Pull request merged**
